### ISSUE TITLE
Local Libraries/Functions Not Visible/Accessible From Code Explorer

### ISSUE THREAD

**Issue by Clowcadia** [maestrith/AHK-Studio#143]
AHK Studio when it comes to code explorer does not act like AHK works.
AHK Studio only searches Standart or/and User Libraries, to display their functions in the Code Explorer, but not the local libraries function like AHK does.

AHK info:
%A_ScriptDir%\Lib\  ; Local library - requires v1.0.90+.
%A_MyDocuments%\AutoHotkey\Lib\  ; User library.
path-to-the-currently-running-AutoHotkey.exe\Lib\  ; Standard library

[v1.0.90+]: The local library is supported and is searched before the user library and standard library.
For example, if a script calls a nonexistent function MyFunc(), the program searches for a file named "MyFunc.ahk" in the user library. If not found there, it searches for it in the standard library.

**Comment by Clowcadia** [comment]
I think the easiest solution is to have all the library paths in an array or something, and just use a for each loop to find all the items for the explorers. Like files, classes and functions.

**Comment by tdalon** [comment]
Yes I also use %A_ScriptDir%\Lib\ and such libraries aren't displayed in AHK Studio unfortunately
